app/views/webpages/tudsc.html.erb

```diff
@@ -10,57 +10,44 @@
 
 <div class="row">
 	<div class="col">
-		<h1 class="page-title text-center">
+		<h1 class="page-title">
 			<%= title t("manifold.webpages.tudsc.page_title") %>
 		</h1>
 	</div>
 </div>
 
 <div class="row">
-
 	<div class="col-12 col-lg-4 pr-14 left-links">
-
-    <%= render "covid_alert_mini" unless @header_alert.nil? %>
-
-		<div class="locations">
-			<%= link_to t("manifold.webpages.tudsc.locations.makerspace_label"), 
+		<ul class="locations list-unstyled">
+      <li>
```

Comment:
@cdoyle-temple I'm not sure if it is your editor, but there is a lot of strange indentation on all of these html.erb files.  For example, the <li> tag is closer to the left than the <ul> tag.  Is there a reason the indentation for these files doesn't follow the usual flow?

Reply:
Hm, doesn't show like that in my editor. Some of these older files must have been saved with a different spaces per tab setting, and my editor must do a good job of making them look nice. Let me see if I can clean it up.

Reply:
The same issue is present in most of the html.erb files.  Were you going to update those as part of this, or just that one?  If it is too much work to do now, I'm okay with you creating a Jira ticket to clean things up at another time.

Reply:
It was not in the plan. I'll open a ticket for it.